unit_tests/testIAPWSWaterProperties/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+#++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+#                          Futility Development Group                          !
+#                             All rights reserved.                             !
+#                                                                              !
+# Futility is a jointly-maintained, open-source project between the University !
+# of Michigan and Oak Ridge National Laboratory.  The copyright and license    !
+# can be found in LICENSE.txt in the head directory of this repository.        !
+#++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
```

Comment:
Do we usually put our headers on cmake files?  I thought it was just .f90 and .C files.  It's not hurting anything, just more of a question on whether we should make this change elsewhere too.  I definitely haven't added these to MPACT cmake files.

Reply:
I"m 99.999% positive I just copied this from another cmake file.